BlazarService/src/main/java/com/hubspot/blazar/guice/BlazarServiceModule.java

```diff
@@ -91,34 +88,33 @@ public void configure(Binder binder) {
     binder.bind(ObjectMapper.class).toInstance(getEnvironment().getObjectMapper());
     Multibinder.newSetBinder(binder, ContainerRequestFilter.class).addBinding().to(GitHubNamingFilter.class).in(Scopes.SINGLETON);
 
+    Multibinder<RepositoryBuildVisitor> repositoryBuildVisitorMultibinder = Multibinder.newSetBinder(binder, RepositoryBuildVisitor.class);
```

Comment:
Why don't you initialize the Multibinders inside the modules that set their values?